test/js/ui/camera.test.js

```diff
@@ -979,21 +979,21 @@ test('camera', (t) => {
         t.test('peaks at the specified zoom level', (t) => {
             const camera = createCamera();
             camera.setZoom(20);
-            let minZoom = Infinity;
+
+            const minZoom = 1;
 
             camera.on('zoom', () => {
-                if (camera.getZoom() < minZoom) {
-                    minZoom = camera.getZoom();
+                const zoom = camera.getZoom();
+                if (zoom < 1) {
```

Comment:
Just confirmed that this test passes if we use the `easeTo` method as well, so adding this check back is important.